### ISSUE TITLE
Tab Accessibility on RoadMap Page Does Not Work Once We Reach Add Year

### ISSUE THREAD

**Issue by SenghoungLim** [icssc/peterportal-client#252]
Using tab to navigate the website. Once we reach add year, after [enter], we cannot tab to add year or change the year from the drop down menu; it just goes straight to the search bar on the right side.
<img src='https://recordit.co/IFmS3rlhmO.gif' title='Video Walkthrough' width='' alt='Video Walkthrough' />
